ui/components/molecules/M003/stories.js

```diff
@@ -3,7 +3,6 @@ import { storiesOf } from '@storybook/react'
 
 import Nav from '.'
 
-storiesOf('Molecules|M003', module)
-.add('Default', () => (
+storiesOf('Molecules|M003', module).add('Default', () => (
   <Nav title={`'First item'`} />
 ))
```

Comment:
Ditto: I need to clean/update or delete this,  Added task in Asana.

Reply:
:+1: